recipes-core/packagegroups/packagegroup-xenclient-dom0.bb

```diff
@@ -16,7 +16,6 @@ RDEPENDS_${PN} = " \
     xen-hypervisor \
     xen-efi \
     xen-flask-tools \
-    xen-libxenctrl \
```

Comment:
No, xen-libxenctrl will still be installed. It is now an RPROVIDES of the xen package.

Reply:
If actual package dependencies are set properly, I don't think there should be any `xen-lib*` entries in this RDEPENDS.  Installing `xen-xl` and the other xen packages should pull in whatever they need.

Did something in the 4.12 uprev require this change?

Reply:
@jandryuk I don't think so, but removing all the `xen-lib*` packages will have the correct dependencies installed. Filed [OXT-1620](https://openxt.atlassian.net/browse/OXT-1620) to track this and another complementary efforts. 

Since this PR is blocking further development and that hunk has introduces no risk, I think this can be addressed later.